frontends/bsd/src/screens/machine_locked_screen.tsx

```diff
@@ -20,7 +20,7 @@ export function MachineLockedScreen(): JSX.Element {
         <MainChild maxWidth={false} centerHorizontal centerVertical>
           <LockedImage src="locked.svg" alt="Locked Icon" />
           <Prose textCenter theme={fontSizeTheme.medium} maxWidth={false}>
-            <h1>Machine Locked</h1>
+            <h1>VxCentralScan is Locked</h1>
```

Comment:
I contemplated moving our lock screens to `@votingworks/ui` as part of this PR but felt that this was out of scope. There's a preexisting GitHub issue ([#893](https://github.com/votingworks/vxsuite/issues/893)) for this code sharing!